opencti-platform/opencti-front/src/private/components/settings/settings_messages/SettingsMessageCreation.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import SettingsMessageForm from './SettingsMessageForm';
+
+const SettingsMessageEdition = ({
```

Comment:
Why is the component called SettingsMessageEdition in SettingsMessageCreation.tsx ? it's then duplicated in SettingsMessageEdition.tsx